docs/content/how-to/powervs/create-cluster-powervs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Create PowerVS Hypershift Cluster
+---
+
+# Create PowerVS Hypershift Cluster
+
+Creating Hypershift cluster in IBM Cloud PowerVS service.
+
+## Prerequisites
+
+Please see [prerequisites](./prerequisites-and-env-guide.md/#prerequisites) before setting up the cluster
+
+## Creating the Cluster
+
+Use the `hypershift create cluster powervs` command:
+
+    ./bin/hypershift create infra powervs --base-domain BASEDOMAIN
+        --resource-group RESOURCE_GROUP \
+        --pull-secret PULL_SECRET \
+        --region POWERVS_REGION \
+        --zone POWERVS_ZONE \
+        --vpc-region VPC_REGION \
+        --node-pool-replicas=2
```

Comment:
I think this should be moved to the `Create cluster with infra created separately` section below, and should include `--output-file` and `--infra-id` ?

Reply:
This should be a example for create cluster command only, fixed now.